bee-message/src/payload/transaction/essence.rs

```diff
@@ -24,10 +24,8 @@ use core::{convert::Infallible, fmt};
 /// Length (in bytes) of Transaction Essence pledge IDs (node IDs relating to pledge mana).
 pub const PLEDGE_ID_LENGTH: usize = 32;
 
-pub(crate) const PREFIXED_INPUTS_LENGTH_MAX: u32 = *INPUT_COUNT_RANGE.end() as u32;
-pub(crate) const PREFIXED_OUTPUTS_LENGTH_MAX: u32 = *OUTPUT_COUNT_RANGE.end() as u32;
-pub(crate) const PREFIXED_INPUTS_LENGTH_MIN: u32 = *INPUT_COUNT_RANGE.start() as u32;
-pub(crate) const PREFIXED_OUTPUTS_LENGTH_MIN: u32 = *OUTPUT_COUNT_RANGE.start() as u32;
+pub(crate) type InputCount = BoundedU32<{ *INPUT_COUNT_RANGE.start() as u32 }, { *INPUT_COUNT_RANGE.end() as u32 }>;
```

Comment:
You sometime put plural and sometime singular before *Count, I suggest using plural. Can you have a look and harmonise them all please ?

Reply:
I kept the name consistent with the constants that were defined before my changes. If the constant said `COUNT`, the type is called `Count`. I can change it if you want nevertheless

Reply:
Would be nice if we could yes. What do you think make the most sense ? Singular or plural ?

Reply:
I'll check it and let you know because I'm not sure.